doc/PLUGINS.md

```diff
@@ -678,14 +678,25 @@ declares that it'd like to be consulted on what to do next for certain
 events in the daemon. A hook can then decide how `lightningd` should
 react to the given event.
 
+When hooks are registered, they can optionally specify "before" and
+"after" arrays of plugin names, which control what order they will be
```

Comment:
General question about 'plugin names': Is this same as for `cli plugin start/stop` commands, meaning either full path (useless for this feature) or basename? If so, we may note that the plugin name includes its extension, meaning full filename.

Reply:
Yes, filename or full path name.